Jenkinsfile

```
@@ -0,0 +1,46 @@
+pipeline {
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '20'))
+        timestamps()
+    }
+    agent {
+        docker {
+            label 'd3-build-agent'
+            image 'openjdk:8-jdk-alpine'
+            args '-v /var/run/docker.sock:/var/run/docker.sock -v /tmp:/tmp'
+        }
+    }
+    stages {
+        stage('Build') {
+            steps {
+                script {
+                    sh "./gradlew build --info"
+                }
+            }
+        }
+        stage('Test') {
+            steps {
+                script {
+                    env.WORKSPACE = pwd()
+                    sh "./gradlew test --info"
+                }
+            }
+        }
+        stage('Build artifacts') {
+            steps {
+                script {
+                    if (env.BRANCH_NAME ==~ /(master|develop)/) {
```

I think we should use a stable `master`  branch for deployments into Gradle
```suggestion
                    if (env.BRANCH_NAME ==~ /(master)/) {
```